### DISCUSSION TITLE
Docker volumes

### DISCUSSION THREAD

**Discussion by 86bjorn** [jvde-github/AIS-catcher#163] [Q&A] [answered]
Hi!

First of all great software! My rtl sdr with an raspberry pi 2 boosted 640% in signals with this upgrade.
My question.
I want to mount the data outside the docker. So the graphs will be saved after a restart. 
Witch folder (debian) are the datas stored?

Thnx!

**Comment by jvde-github** [comment, marked as answer]
Hey, thanks! Glad to hear.

The backup is not created by default but will be with the setting `-N FILE filename`. So, free choice where to put it. On the docker command line you can map it to a file outside the container:
```
docker run --rm -it --pull always -v /home/jasper:/opt --device /dev/bus/usb -p 8100:8100 ghcr.io/jvde-github/ais-catcher:latest -N 8100 file /opt/backup
```
Perhaps you can keep the backup file in the container without mapping it locally, but have not tested that.

Does that answer it?

**Comment by 86bjorn** [comment]
Hi!

I get an Error after i guess the one minute backup time. 'ais-catcher exited with code 255'
My docker-compose setup.
`ais-catcher:
    command: -v 60 -q -u 127.0.0.1 10110 -N 8100 -N file /opt/backup/stat.bin BACKUP 1 -p 59
    container_name: ais-catcher
    devices:
      - /dev/bus/usb
    image: ghcr.io/jvde-github/ais-catcher:latest
    ports:
      - "8100:8100"
    restart: always
    environment:
      DISABLE_IPV6: 'true'
      TZ: Europe/Amsterdam
    volumes:
      - "./ais-catcher/data:/opt/backup"`

**Reply by 86bjorn** [reply]
-N file /opt/backup/stat.bin BACKUP 1 without the backup 1 seems to work. (-N file /opt/backup/stat.bin)

**Reply by jvde-github** [reply]
Minimum is 5 minutes, e.g. BACKUP 5. Sorry poorly documented and poor error messaging. Will add some clarifications tonight.

**Reply by jvde-github** [reply]
I have included a more informative error message in the program (now at least mentions the parameter). Also documentation is updated.
